README.md

```diff
@@ -82,6 +82,12 @@ git push --set-upstream origin master
 ```
 git status
 ```
+
+* Git abort changes in a file
+
+```
+git checkout file_name.py
```

Comment:
This record doesn't always work. It's better to use:
`git checkout -- path/to/file.py`